sources/packages/forms/src/form-definitions/educationprogram.json

```diff
@@ -563,6 +563,10 @@
                                 "label": "Undergraduate Certificate",
                                 "value": "undergraduateCertificate"
                             },
```

Comment:
The value `undergraduateCitation` is missing to be added to the hidden variable `Level Of Study Codes`

![image](https://github.com/bcgov/SIMS/assets/54600590/8509ff64-d896-445a-a2bb-048370a1500e)

Reply:
Sorry, I did that but then I messed this a bit. Thanks!